packages/react-components/react-card/src/components/Card/useCardStyles.ts

```diff
@@ -6,24 +6,28 @@ import { cardFooterClassNames } from '../CardFooter/useCardFooterStyles';
 import type { CardSlots, CardState } from './Card.types';
 import type { SlotClassNames } from '@fluentui/react-utilities';
 
+/**
+ * Static CSS class names used internally for the component slots.
```

Comment:
tip: you can mark it `@internal` . after we implement internal public API removals this will "just work"

Reply:
I think this is one of the cases where we do want to export this to prevent users who need to apply styles to the static class names from depending on a hardcoded class name.